openssh.advisories.yaml

```diff
@@ -6,3 +6,8 @@ advisories:
     - timestamp: 2023-02-06T13:39:35.322093-05:00
       status: fixed
       fixed-version: 9.2_p1-r0
+
+  CVE-2023-38408:
+    - timestamp: 2023-07-19T11:06:22.022093-05:00
+      status: fixed
+      fixed-version: 9.2_p2-r0
```

Comment:
Just double-checking this version. Your [other PR](https://github.com/wolfi-dev/os/pull/3706/files#diff-3f462f3caa4e52cd56c66134398413393ad090528f27872394128f08b785ec12R3) set the version to `9.3_p2`

Reply:
good catch - fixed!